bundles/org.openhab.core.automation.module.script.rulesupport/src/test/java/org/openhab/core/automation/module/script/rulesupport/loader/AbstractScriptFileWatcherTest.java

```diff
@@ -47,22 +53,26 @@
  * @author Jonathan Gilbert - initial contribution
  */
 @NonNullByDefault
-class ScriptFileWatcherTest {
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+class AbstractScriptFileWatcherTest {
 
-    private @NonNullByDefault({}) ScriptFileWatcher scriptFileWatcher;
-    private @NonNullByDefault({}) ScriptEngineManager scriptEngineManager;
-    private @NonNullByDefault({}) DependencyTracker dependencyTracker;
+    private @NonNullByDefault({}) AbstractScriptFileWatcher scriptFileWatcher;
+
+    @Mock
+    private @NonNullByDefault({}) ScriptEngineManager scriptEngineManagerMock;
```

Comment:
There will be less vertical scrolling when adding the annotation on the same line.

```suggestion
    private @Mock @NonNullByDefault({}) ScriptEngineManager scriptEngineManagerMock;
```